### DISCUSSION TITLE
Milvus Intergration with Databricks (or Spark in general)

### DISCUSSION THREAD

**Discussion by viv1729** [milvus-io/milvus#6909] [Q&A and General discussion]
Hi,
What would be the best way to integrate Milvus with Databricks (if possible). Any suggestions or workarounds would be helpful.

**Comment by xiaofan-luan** [comment]
> Hi,
> What would be the best way to integrate Milvus with Databricks (if possible). Any suggestions or workarounds would be helpful.
This is a good topic? can you specify more details about what you want to do?
Are you using spark streaming or ML Flow? what kind of integration do you want? We are also thinking about this

**Comment by codingjaguar** [comment]
Circling back here, the integration of Milvus and Databricks / Apache Spark is ready for use now: https://milvus.io/docs/integrate_with_spark.md

Let us know if you have any questions!
